packages/server/src/migrations/data/types.ts

```diff
@@ -31,14 +32,24 @@ export interface PostDeployMigration<T extends PostDeployJobData = PostDeployJob
   run(repo: Repository, job: Job<T> | undefined, data: T): Promise<PostDeployJobRunResult>;
 }
 
-// Custom Jobs
+export type MigrationActionResult = { name: string; durationMs: number };
```

Comment:
What was the motivation for renaming `CustomMigrationAction` to `MigrationActionResult`?

It seems that we're now being somewhat loose with "action" vs "result", which seems like distinct concepts?

Sorry, I hate to be pedantic, but it seems like this is a big part of the diff, and it's not totally clear.

Naming things is hard.

Reply:
Yeah...I think normalizing on "action" for the things to be done in a migration and "action result" or "results" as the outcomes from those things.

Pushing a lot more diff to align with that.